examples/axisartist/demo_curvelinear_grid2.py

```diff
@@ -1,6 +1,6 @@
 """
 ======================
-Demo Curvelinear Grid2
+Demo Curvilinear Grid2
```

Comment:
This one is on purpose; see the filename and classes used. Maybe it should be `CurveLinear` in the title instead, though?